doc/search-metrics.org

```diff
@@ -134,13 +144,17 @@ GET /ctia/incident/histogram?aggregate-on=timestamp&granularity=day&status=Open&
 #+END_SRC
 it will return for instance
 #+BEGIN_SRC javascript
-[{"key": 2020-01-01", "value": 10},
- {"key": 2020-01-02", "value": 0},
- {"key": 2020-01-03", "value": 6},
- {"key": 2020-01-04", "value": 2},
- {"key": 2020-01-05", "value": 0},
- {"key": 2020-01-06", "value": 1},
- {"key": 2020-01-07", "value": 6},
- {"key": 2020-01-08", "value": 4},
- {"key": 2020-01-09", "value": 3}]
+{"data": {"timestamp": [{"key": "2020-01-01", "value": 10},
```

Comment:
here there is not value and I think this is fine.

Reply:
here we aggregate on `timestamp`